src/site/resources/styleguides/google-java-style-20180523/javaguide.html

```diff
@@ -727,14 +750,37 @@ <h5 id="s4.8.4.3-switch-default">4.8.4.3 The <code>default</code> case is presen
 <a name="annotations"></a>
 <h4 id="s4.8.5-annotations">4.8.5 Annotations</h4>
 
-<p>Annotations applying to a class, method or constructor appear immediately after the
+<h5 id="s4.8.5.1-type-use-annotation-style">4.8.5.1 Type-use annotations</h5>
+
+<p>Type-use annotations appear immediately before the annotated type. An annotation is a type-use
+annotation if it is meta-annotated with
+<code class="prettyprint lang-java">@Target(ElementType.TYPE_USE)</code>. Example:</p>
+
+<pre class="prettyprint lang-java">final @Nullable String name;
+
+public @Nullable Person getPersonByName(String name);
+</pre>
```

Comment:
this need to be investigated.

Reply:
please share link to your comment.

Reply:
https://github.com/checkstyle/checkstyle/pull/14901/files#r1620639191

Reply:
@Zopsss could you create an issue for this, we might need to go with a blue check and add description with checkstyle limitation to catch such cases. Reply to this comment with a link to the created issue.

Reply:
@Zopsss , please create issue on this.

Reply:
issue opened: https://github.com/checkstyle/checkstyle/issues/15416

I think we cannot follow this rule at all as we need to check if the annotation was created using `@Target(ElementType.TYPE_USE)`, I have explained the reason more in detail in the issue.

Reply:
coverage update is at https://github.com/checkstyle/checkstyle/pull/15419

thread is resolved.